tests/functional/p/postponed_evaluation_pep585.txt

```diff
@@ -0,0 +1,12 @@
+unsubscriptable-object:14:15::Value 'list' is unsubscriptable
```

Comment:
Probably a nitpick, but it seems that the column should be the current value +1 to be the l of `list`.

Reply:
I used the test script to generate them.
```
python tests/test_functional.py --update-functional-output -k "test_functional[postponed_evaluation_pep585]"
```
Maybe they start counting at 0 instead of 1?

Reply:
The script is using the current output of pylint. Something has to be fixed in the code but this was a pre-existing bug, with nothing to do with your current change. Don't bother, will fix in another MR :)